core/src/main/java/org/arquillian/smart/testing/Logger.java

```diff
@@ -0,0 +1,163 @@
+package org.arquillian.smart.testing;
+
+import java.util.logging.Level;
+
+import static java.lang.String.format;
+
+public class Logger {
+
+    private java.util.logging.Logger jul;
+
+    private static String prefix = "%s: Smart-Testing - ";
+
+    private Logger(java.util.logging.Logger logger) {
+        this.jul = logger;
+    }
+
+    /**
+     * Find or create a logger for a named subsystem.
+     *
+     * @param clazz
+     *     A class name of the subsystem for the logger such as java.net.URI or javax.swing.Box
+     *
+     * @return a suitable Logger
+     */
+    public static Logger getLogger(Class clazz) {
+        return getLogger(clazz.getName());
+    }
+
+    /**
+     * Find or create a logger for a named subsystem using java.util.logging.Logger.
+     * If a logger has already been created with the given name it is returned.
+     * Otherwise a new logger is created.
```

Comment:
Does this description really holds to what the code actually does? I think it is only true for the jul logger we will be delegating to?